config/applets/dump_klv.conf

```diff
@@ -0,0 +1,4 @@
+# Default configuration for core dump-klv applet
+
+video_reader:type = ffmpeg
+video_reader:vidl_ffmpeg:time_source = misp
```

Comment:
`time_source` was copied from the hard coded default config, but this no longer does anything unless the reader type is set to `vidl_ffmpeg` instead of `ffmpeg`.